includes/api/order.php

```diff
@@ -12,6 +12,16 @@ function createWcOrder(WP_REST_Request $request)
     $logObj           = array();
     $logObj['api']    = 'createWcOrder';
     $logObj['params'] = $params;
+
+    // fetching wp_woocommerce_session_ from cookies
+    $sessionVal = array_filter($params['cookies'], function($key) {
+       return strpos($key, 'wp_woocommerce_session_') === 0;
+    }, ARRAY_FILTER_USE_KEY);
+
+    foreach($sessionVal as $key => $value){
```

Comment:
@poojarazor did we verify the latency added by this logic?

Reply:
yes , verified not adding any latency